components/ResultsPageComponents/SharedComponents/ContentStyles.jsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@chakra-ui/react";
+
+const ResultContentSection = (props) => {
```

Comment:
Can we align with the convention that filenames of React components should be the same as the component name please?
Please either:
- rename the file from `ContentStyles.jsx` to `ResultContentSection.jsx`, or
- rename the component to `ContentStyles` 

I think renaming the file makes more sense.

Reply:
I was going to have more than one component defined in that file. If you prefer one component per file, I'm happy to change this.

Reply:
Let's go with one file per component.

- it's the most common pattern we are using to organise code in our codebase (although there might be exceptions here and there...)
- it might also make it easier to write tests for components in future if they are in separate files